src/styles/blocks/contacts.scss

```diff
@@ -0,0 +1,34 @@
+.contacts {
+  &__field {
+    margin-bottom: 16px;
+
+    &--text {
+      margin-bottom: 30px;
+    }
+  }
+
+  &__info {
+    margin-bottom: 24px;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  &__info-title {
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 20px;
+    color: #9fadc2;
+    margin-bottom: 8px;
+  }
+
+  &__info-link {
+    text-decoration: none;
```

Comment:
Try to group your styles according this recommendation  https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/ 
It will improve the code readability
Please check your styles and refactor according this recommendation

Reply:
есть)